docs/setup-development.md

```diff
@@ -0,0 +1,235 @@
+
+`
+python3 manage.py runserver -d -r
+`
+
+You can access the API documentation on [http://localhost:5000/api-docs](http://localhost:5000/api-docs), it also allows you to execute requests on your local TM instance. The API docs is also available on our [production](https://tasks.hotosm.org/api-docs/) and [staging](https://tasks-stage.hotosm.org) instances.
+
+#### API Authentication
+
+In order to authenticate on the API, you need to have an Authorization Token.
+
+1. Run the command line `manage.py` with the `gen_token` option and `-u <OSM_User_ID_number>`. The command line can be run in any shell session as long as you are in the tasking-manager directory.
+
+```
+venv/bin/python manage.py gen_token -u 99999999
+```
+
+This will generate a line that looks like this:
+> Your base64 encoded session token: b'SWpFaS5EaEoxRlEubHRVC1DSTVJZ2hfalMc0xlalu3KRk5BUGk0'
+
+2. In the Swagger UI, where it says
+> Token sessionTokenHere==
+
+replace `sessionTokenHere==` with the string of characters between the apostrophes (' ') above so you end up with something that looks like this in that field:
+
+> Token SWpFaS5EaEoxRlEubHRVC1DSTVJZ2hfalMc0xlalu3KRk5BUGk0
+
+Your user must have logged in to the local testing instance once of course and have the needed permissions for the API call.
+
+You can get your OSM user id number either by finding it in your local testing/dev database `select * from users` or from OSM by viewing the edit history of your user, selecting a changeset from the list, and then at the bottom link `Changeset XML` and it will be in the `uid` field of the XML returned.
+
+To get your token on the production or staging Tasking Manager instances, after sign in in the browser, inspect a network request and search for the `Authorization` field in the request headers section.
+
+### DevOps
+If you encounter any issues while setting up a dev environment, please visit our [FAQ ❓ page](./setup-development-faqs.md) to find possible solutions.
+
+# Docker
+
+## Creating a local PostGIS database with Docker
+
+If you're not able to connect to an existing tasking-manager DB, we have a [Dockerfile]() that will allow you to run PostGIS locally as follows.
+
+### Dependencies
+
+Following must be available locally:
+
+* [Docker CE](https://www.docker.com/community-edition#/download)
+
+### Build & Run the PostGIS dockerfile
+
+1. From the root of the project:
+
+`
+docker build -t tasking-manager-db ./devops/docker/postgis
+`
+
+2. The image should be downloaded and build locally.  Once complete you should see it listed, with
+
+`
+docker images
+`
+
+3. You can now run the image (this will run PostGIS in a docker container, with port 5432 mapped to localhost):
+
+`
+docker run -d -p 5432:5432 tasking-manager-db
+`
+
+4.  Confirm the image is running successfully:
+
+`
+docker ps
+`
+
+5. Finally you can set your env variable to point at your containerised DB:
+
+`
+export TM_DB=postgresql://hottm:hottm@localhost/tasking-manager
```

Comment:
I think that TM_DB variable is outdated. On the top of the page it recommends another set of env vars.

Reply:
`TM_DB` still works, but the other ones should be mentioned here. However this PR is more about moving the pages in. I agree with you that we have to go through the whole documentation and update it. This PR can only be a first step.